vars/Kali.yml

```diff
@@ -0,0 +1,36 @@
+---
+# The Linux distribution to use when configuring a Debian repo.  Note
+# that we force Kali to use Debian Bookworm.
+apt_distro: debian
+
+# The release of the Linux distribution to use when configuring a
+# Debian repo.  Note that we force Kali to use Debian Bookworm.
+apt_distro_release: bookworm
```

Comment:
Can you please add a brief comment here explaining why we are forcing Kali to use Debian Bookworm?

Reply:
Please see commit 4854696.

Reply:
Thanks from me and all of our future selves.